converter/.gitignore

```diff
@@ -1,3 +1,4 @@
 DefaultRun.json
 DefaultRun.nodes
-src/main/java/gov/cms/qpp/conversion/correlation/model/*
\ No newline at end of file
+src/main/java/gov/cms/qpp/conversion/correlation/model/*
+/bin/
```

Comment:
Ignore `/bin/`?

Reply:
This happened automatically. I can remove it, but I guess a bin folder popped up and got ignored